src/user/user.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { UserModel } from 'src/models/user.model';
+
+@Injectable()
+export class UserService {
+    constructor(
+        @InjectModel(UserModel) private userModel: typeof UserModel
```

Comment:
Bravo ! Search more about a concept named "dependency injection" to know more about this line.